### DISCUSSION TITLE
Linux

### DISCUSSION THREAD

**Discussion by Chaolegio01** [masterking32/MasterHamsterKombatBot#17] [Q&A] [answered]
please help me this

/MasterHamsterKombatBot$ py main.py
Traceback (most recent call last):
  File "/usr/bin/py", line 167, in <module>
    result = eval(args.expression)
  File "<string>", line 1, in <module>
    main.py
NameError: name 'main' is not defined

**Comment by masterking32** [comment, marked as answer]
try with `python3 main.py`

**Comment by Chaolegio01** [comment]
Ty! for your answer. But stiill get this error.

python3 main.py
Traceback (most recent call last):
  File "/home/ubuntu/main.py", line 13, in <module>
    from utilities import (
ModuleNotFoundError: No module named 'utilities'
ubuntu@ip-172-31-24-1:~$ python3
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.

**Comment by Chaolegio01** [comment]
Sorry my bad. Not cd to folder. i can run it now. Thank you for your support !
